content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,150 @@
+</div>
+
+This page requests a CSS file with eight classes, but not all of them are necessary to render the "visible" content. 
+
+The goal of this guide is to optimize this page, so only the **critical** styles are loaded synchronously, while the rest (like the ones applied to paragraphs), are loaded in a non-blocking way.
+
+## Measure
+
+Run [Lighthouse](https://web.dev/fast/discover-performance-opportunities-with-lighthouse/#run-lighthouse-from-chrome-devtools) on [the page](https://defer-css-unoptimized.glitch.me/) and go to the **Performance** section.
+
+The report shows the **FCP** metric with a value of "1s", and the opportunity **Eliminate render-blocking resources**, pointing to the **style.css** file:
+
+<figure>
+  <img src="./lighthouse-unoptimized.png"
+       alt="Lighthouse report for unoptimized page, showing FCP of '1s' and 'Eliminate blocking resources' under 'Opportunities'" class="screenshot">
+</figure>
+
+To visualize how this CSS is actually blocking rendering:
+
+1. Open [the page](https://defer-css-unoptimized.glitch.me/) in Chrome.
+1. Press Control+Shift+J or Cmd+Option+J (Mac), to open DevTools.
+1. Go to the **Performance** tab and click on the **Reload** button.
+
+In the resulting trace you’ll see that the **FCP** marker is placed immediately after the CSS finishes loading:
+
+<figure>
+  <img src="./cdt-perf-unoptimized.png"
+       alt="DevTools performance trace for unoptimized page, showing FCP starting after CSS loads." class="screenshot">
+</figure>
+
+This means that the browser needs to wait for the whole CSS to be loaded and processed before painting a single pixel on the screen.
+
+## Optimize
+
+To optimize this page, you need to know which classes are considered “critical”.
+You'll use the the [Coverage Tool](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) for that:
+
+1. In DevTools, open the [Command Menu](https://developers.google.com/web/tools/chrome-devtools/ui#command-menu), by pressing Control+Shift+P or Cmd+Shift+P (Mac).
+1. Type "Coverage" and select "Show Coverage".
+1. Click the "Reload" button, to reload the page and start capturing the coverage.
+
+<br/>
+
+<figure>
+  <img src="./coverage-unoptimized.png"
+       alt="Coverage for CSS file, showing 55.9% unused bytes." class="screenshot">
+</figure>
+
+
+Double-click on the report, to see classes marked in two colors:
+
+* Green (**critical**): These are the classes the browser needs to render the visible content (like the title, subtitle, and accordion buttons).
+* Red (**non-critical**): These styles are applied to content that's not immediately visible (like the paragraphs that appear after clicking on each button). Additionally, class definitions like "h2" are not even used on this page.
+
+With this information, you'll apply an optimization to let the browser start processing the critical styles immediately after the page loads, while deferring the load of the non-critical CSS for later:
```

Comment:
A little learning... "With this information, optimize your CSS so that the browser starts processing critical styles immediately after page loads, while deferring non-critical CSS for later:"